relayer-cli/src/commands/query/packet.rs

```diff
@@ -224,32 +190,34 @@ impl Runnable for QueryUnreceivedPacketsCmd {
             pagination: ibc_proto::cosmos::base::query::pagination::all(),
         };
 
-        let seq_res = counterparty_chain
+        let commitments = counterparty_chain
             .query_packet_commitments(commitments_request)
-            .map_err(|e| Kind::Query.context(e));
+            .map_err(|e| Kind::Query.context(e))?;
 
         // extract the sequences
-        let sequences: Vec<u64> = match seq_res {
-            Ok(seqs) => seqs.0.into_iter().map(|v| v.sequence).collect(),
-            Err(e) => {
-                return Output::error(format!(
-                    "failed to fetch the packet commitments from chain ({}) with error: {}",
-                    chain.id(),
-                    e
-                ))
-                .exit()
-            }
-        };
+        let sequences: Vec<u64> = commitments.0.into_iter().map(|v| v.sequence).collect();
+
+        trace!(
```

Comment:
This sounds more like debug information than trace:

```suggestion
        debug!(
```